src/Optimizely/OptimizelyConfig/OptimizelyConfigService.php

```diff
@@ -207,6 +311,81 @@ protected function getVariationsMap(Experiment $experiment)
         return $variationsMap;
     }
 
+    /**
+     * Generates string of audience conditions mapped to audience names.
+     *
+     * @param array experiment audience conditions .
+     *
+     * @return string of experiment audience conditions.
+     */
+    protected function getAudiences(array $audienceCond)
+    {
+        $projectConfig = $this->projectConfig;
+        $resAudiences = '';
+        $audienceConditions = array('and', 'or', 'not');
+        if ($audienceCond != null){
+            $cond = '';
+            foreach($audienceCond as $var) {
```

Comment:
Please revise this method according to C# PR: https://github.com/optimizely/csharp-sdk/pull/266